### DISCUSSION TITLE
Vercel custom domain not working

### DISCUSSION THREAD

**Discussion by Dawid2806** [vercel/community#2238] [Help]
Hi, I bought a domain in vercel but I keep getting an error

![image-png-attachment (2).png](https://user-images.githubusercontent.com/95871163/235313226-fdf54f17-ab70-44ea-8a6d-95fe2a279e3e.png)

![image-png-attachment (1).png](https://user-images.githubusercontent.com/95871163/235313229-8c1bd539-90ec-408b-803c-1fbb5ecac263.png)

![image-png-attachment.png](https://user-images.githubusercontent.com/95871163/235313231-f2a4d6a6-5abf-49a6-8419-0885c5bc193f.png)

**Comment by smaeda-ks** [comment]
@Dawid2806 do you still need help? seems like you figured it out
